src/pages/Details/Details.js

```diff
@@ -1,8 +1,89 @@
 import React, { Component } from 'react';
 import './Details.scss';
 
+import AddToCart from './AddToCart';
+import Brief from './Brief';
+
 export default class Details extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      infoList: [],
+      hashtag: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/DetailTest2.json')
+      .then(res => res.json())
+      .then(res => this.setState({ infoList: res }));
+  }
+
   render() {
-    return <div>Hello, Details!</div>;
+    return (
+      <div className="details-wrapper">
+        <div className="product-info">
+          {this.state.infoList.map(
+            ({
+              id,
+              name,
+              price,
+              ml,
+              tiny_description,
+              dgree,
+              expire_date,
+              keep,
+              category_name,
+              image,
+              hash,
+            }) => {
+              return (
+                <Brief
+                  key={id}
+                  name={name}
+                  price={price}
+                  size={ml}
+                  dscr={tiny_description}
+                  degree={dgree}
+                  expireDate={expire_date}
+                  keep={keep}
+                  category={category_name}
+                  mainImage={image}
+                  hashtag={hash}
+                />
```

Comment:
```suggestion
          {this.state.infoList.map(
            (product) => (
                <Brief
                  key={product.id}
                  name={product.name}
                  price={product.price}
                  size={product.ml}
                  dscr={product.tiny_description}
                  degree={product.dgree}
                  expireDate={product.expire_date}
                  keep={product.keep}
                  category={product.category_name}
                  mainImage={product.image}
                  hashtag={product.hash}
                />
```
- 이 경우에는 구조분해 할당이 오히려 가독성에 좋지 않아서 이렇게 써주시는게 더 좋겠습니다
- infoList에서 꺼내오는 값은 info라고 볼 수 있는데 이 네이밍 자체로는 어떤 값인지 알수 없어서 product 같은 구체적인 이름으로 바꿔주면 좋겠네요.

Reply:
넵 확인했고 리뷰 사항 반영하여 수정하였습니다 😊👌🏽